tests/test_wave_energy.py

```diff
@@ -11,7 +11,7 @@
 # various time steppers.
 
 
-def wave(n, deg, butcher_tableau, splitting=AI):
+def wave(n, deg, butcher_tableau, splitting):
```

Comment:
This is a reasonable change, but not really related to this PR...

Reply:
The options to `wave` have been reworked a good bit to allow us to test GL and PEPRK methods with the same suite.